tests/fix_mode_strict/BUILD.in

```diff
@@ -0,0 +1,4 @@
+go_library(
+  name = "syntax_error_no_comma"
+  visibility = ["//visibility:public"],
```

Comment:
What happens if there are multiple errors (maybe even across multiple files)? Do they accumulate and all get printed out? I presume this is mostly going to be used in CI pipelines, should we try to get all of the errors printed out before quitting?

Reply:
It exits at the first error. I give that a quick thought as well and decided not to do it. 

The bzl parser bailed out at the first encountered error, so we can report at most one error per file. Also, the current interface/API doesn't return errors, so to collect all the errors, some serious refactor needs to take place first. See the discussion in #1029.

Reply:
I guess the question is, would it be better to "accumulate" to `stderr` and have a single boolean tracking success and `os.Exit`ing essentially in the `main` function?

Reply:
It would be better to accumulate all errors, but there is no clean way to do so with the current API. I've tried this approach in https://github.com/bazelbuild/bazel-gazelle/compare/master...dr-dime:cl/strict-mode-2, which doesn't LGTM as the error handling strategy is not clear at all.

I admit that those `log.Fatal`s are not pretty, but it's following the comment on error handling in `lang.Language` interface. I can put some big TODOs next to them to make this more explicit as a refactor goal in the future.